### ISSUE TITLE
Tag does not exist

### ISSUE THREAD

**Issue by JC5** [tobybatch/kimai2#241]
It seems the tag `apache-latest` from the readme does not exist.

**Comment by tobybatch** [comment]
Thanks, I'll update the README, I've been trying to simplify the tagging. The latest is now implicity. Just use :apache and you'll get the latest stable release.

**Comment by JC5** [comment]
Thanks, I appreciate it. I saw another discrepancy but I'm not sure if it's a problem.

[This page](https://www.kimai.org/documentation/docker.html) gives the instruction to build the container, which in itself is fine. However, it suggests the tag `kimai/kimai2:dev` which I personally consider your "personal" namespace so it's weird to build under that tag.

Rather, I'd advise to make people build under "kimai_local" or something similar to reflect that it's really their own build. Or just skip the build step since you provide images that work just fine 👍🏻 

The reason I'd avoid the tag `kimai/kimai2:dev` specifically is because it already exists. I skipped the build step (dumb, I know) and inadvertently pulled a 2 year old image from Docker Hub. Imagine my surprise when I ran `apache` 😁
